src/worker/generate-metadata.ts

```diff
@@ -0,0 +1,18 @@
+interface Metadata {
+  license: string;
+  languages: string[];
+}
+
+/**
+ * Given languages, generate a .kmp json file which contains metadata for .kmp zip archive.
+ */
+export function generateMetadata(languages: string[]): string {
+  const license = "mit";
+  const metadata: Metadata = { license, languages };
+
+  return JSON.stringify(metadata);
+}
+
+export function generateMetadataJson(content: string): Blob {
```

Comment:
Any recommendations on how to test Blob? I don't want to do weird juggling and convert it ArrayBuffer or something

Reply:
Unit test should be enough in this case. Since **generateMetadataJson** is just a function that return a value, I would isolate the dependencies by providing a mock string into content and test the return value.